include/seqan3/argument_parser/argument_parser.hpp

```diff
@@ -826,22 +826,23 @@ class argument_parser
      */
     void verify_identifiers(char const short_id, std::string const & long_id)
     {
-        auto constexpr allowed = is_alnum || is_char<'_'> || is_char<'@'>;
+        std::string const valid_chars{"@_0123456789abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ"};
```

Comment:
whats the advantage of making it a class member if it is only used here? I'd rather not bloat the members unnecessarily :shrug:. 
It might be more performant on multiple calls of verify_identifiers.. do you think it's worth it?

Reply:
Yeah, it would only make sense if you called this function multiple times.

But I think `std::array` might still be nice, in lieu of having a `constexpr std::string`. stack vs. heap allocation :)

Reply:
Ok :+1:

Reply:
std::string_view for the win. It's constexpr.

Reply:
:face_palm: thanks